lib/python/ray/worker.py

```diff
@@ -652,13 +652,30 @@ def get_address_info_from_redis_helper(redis_address, node_ip_address):
       elif info[b"client_type"].decode("ascii") == "photon":
         local_schedulers.append(info)
   # Make sure that we got at one plasma manager and local scheduler.
-  assert len(plasma_managers) == 1
-  assert len(local_schedulers) == 1
+  assert len(plasma_managers) >= 1
+  assert len(local_schedulers) >= 1
```

Comment:
I asserted `== 1` because I don't think we are able to handle multiple local schedulers per node in the multinode setting right now. Or are we?